src/glass/src/app/shared/components/language-button/language-button.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+import { getCurrentLanguage, setCurrentLanguage, supportedLanguages } from '~/app/i18n.helper';
+
+@Component({
+  selector: 'glass-language-button',
+  templateUrl: './language-button.component.html',
+  styleUrls: ['./language-button.component.scss']
+})
+export class LanguageButtonComponent {
```

Comment:
IMHO I'm in favor of seeing such additions in separate commits. I think it's fine to add them as part of the PR but maybe separated from the changes adding the join cluster page. I know it's bikeshedding too, so please leave it as it is for this PR... and take it just as an idea for further ones.